### DISCUSSION TITLE
Add Instrument Serif and Reenie Beanie Font

### DISCUSSION THREAD

**Discussion by safire3002** [inkstitch/inkstitch#2517] [Ideas]
Hi all,

I think I've gotten these two fonts in a good place and would like to add them to the font repository. I was wondering if anyone wanted to experiment with stitch-out on their machines though. Please ignore my bad stabilizing in my Reenie Beanie font lol

![IMG_1220](https://github.com/inkstitch/inkstitch/assets/134817062/199798dd-02fa-42dc-82f9-3bc764bbc13b)
![IMG_0849](https://github.com/inkstitch/inkstitch/assets/134817062/b716fcf0-67e2-4f9d-bf3a-537179db288d)

[Instrument Serif.zip](https://github.com/inkstitch/inkstitch/files/12644081/Instrument.Serif.zip)
[Reenie Beanie.zip](https://github.com/inkstitch/inkstitch/files/12644082/Reenie.Beanie.zip)

**Comment by lexelby** [comment]
These look awesome!!  ... And I have so many questions about that preview text for Instrument Serif :o

**Reply by safire3002** [reply]
Thank you! And I told my partner I would stitch any phrase he wants. He chose... that lol

**Comment by bkmgit** [comment]
Thanks. Will get a stitch out in the next few days for ASCII initially.
Can you put them in  a branch/fork  GitHub?
Most font folders do not have spaces, they use underscores.
In the descriptive text, it is helpful to have the height of capital M.

Instrument Serif seems to have kerning pairs that include diacritics.
Probably these need to be done individually.

**Reply by safire3002** [reply]
@bkmgit Thank you for the helpful info. I don't have diacritics built-in but I know I have all the kerning pairs in the json file. I'm not sure how to target those to get deleted. Unless someone wants to add them into my file.

**Comment by bkmgit** [comment]
@safire3002 They came out ok
![IMG_20230925_143603_HDR](https://github.com/inkstitch/inkstitch/assets/1679477/9b9574ab-305f-46e6-871b-45c2af4d37c7)
![IMG_20230925_143549_HDR](https://github.com/inkstitch/inkstitch/assets/1679477/31b229cf-52d3-460c-9820-b885f4e9c632)

**Reply by safire3002** [reply]
@bkmgit Does that mean it's good to add? I've never made a branch or fork before so I'm kind of afraid to append the code.

**Reply by lexelby** [reply]
Oh definitely, we'd love to have your fonts.  Don't worry, we'll be gentle and guide you through the process if you get stuck!

**Comment by bkmgit** [comment]
Perhaps start using the web interface in GitHub. You can then learn to
use the command line if you want to make more contributions.  Fork the
repository. Create directories instrument-serif and reenie-beanie in a
branch of your fork in the fonts directory.  Then upload your files and
make a pull request.  The addresses for the directories in your fork
will be something like:

https://github.com/safire3002/inkstitch/tree/nice-new-fonts/fonts/instrument-serif/
https://github.com/safire3002/inkstitch/tree/nice-new-fonts/fonts/reenie-beanie/


Some tutorials:
https://docs.github.com/en/get-started/quickstart/hello-world
https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests

**Reply by safire3002** [reply]
@bkmgit I think I did it?

**Reply by kaalleen** [reply]
Sorry, I somehow always miss out on the discussions.

For everyone wanting to contribute in future and finds this discussion...
For testing purposes we prefer if this wasn't done from a fork, but from the Ink/Stitch repository directly. Then there will be a development build available which will be easy to use also for non-programmers and they can give their feedback as well. Please contact us and we will be happy to add you to the Ink/Stitch community.
